sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -280,6 +293,7 @@ def test_recursion_decision_tree_vs_forest_and_gbdt(seed):
         np.testing.assert_allclose(pdp_forest, pdp_tree)
 
 
+@pytest.mark.filterwarnings("ignore:A Bunch will be returned")
```

Comment:
instead of adding the filter. In most of cases, we only need `kind="average"`. Like this we have no warning, we test what is intended, and we have the new behaviour. This will apply here and in most of subsequent filtering.

Reply:
Makes sense. Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/7438724ab1e0f844bd79e2a5b0a01e18aac0ca71